modules/ROOT/pages/administration/databases.adoc

```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
 | BOOLEAN
 
 | requestedStatus
-| The expected status of the database. label:default-output[]
+| The expected status of the database. The value can be either `online` of `offline`. label:default-output[]
 | STRING
 
 | currentStatus
 | The actual status of the database. label:default-output[]
+
+The value can be one of the following:
+
+* `online` - The database is running.
+* `offline` - The database is not running.
+* `starting` - The database is not running, but is about to be.
+* `stopping` - The database is not running anymore, but still not stopped completely.
+No offline operations (e.g. `load`/`dump`) can be performed yet.
+* `store copying` - The database is currently being updated from another instance of Neo4j.
+* For other error states please refer to the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/errors/#database-management-states[operation manual].
```

Comment:
I'd maybe drop the "error" here, I'm being pedantic but it sort of implies that the states above are also error states